sys-utils/mountpoint.c

```diff
@@ -182,18 +192,18 @@ int main(int argc, char **argv)
 
 	ctl.path = argv[optind];
 
-	if (stat(ctl.path, &ctl.st)) {
+	if (lstat(ctl.path, &ctl.st)) {
```

Comment:
Seems incorrect for non-nofollow use-case. What about:

rc = ctl.nofollow ? lstat(ctl.path, &ctl.st) : stat(ctl.path, &ctl.st);
if (rc != 0) {

Reply:
Ok, I will do this without relying on lstat() by change working in our favor and doing the right thing in both cases.